www/store/features/drawer.js

```diff
@@ -1,5 +1,5 @@
 const initialState = {
-  open: false
+  open: true
```

Comment:
Why would we want this to be open? I don't think you understand how this is supposed to work.